File: src/OnPeople/BackEnd/src/OnPeople.API/Models/Empresas/Empresa.cs
namespace OnPeople.API.Models.Empresas
{
    public class Empresa
    {
        public int EmpresaId { get; set; }  
        public string NomeEmpresa { get; set; }
        public string NomeFantasia { get; set; }
        public string SiglaEmpresa { get; set; }
        public Boolean Ativa { get; set; }
        public string DataCadastro { get; set; }
        public string DataDesativacao { get; set; }
        public Boolean  Filial { get; set; }
        public int MatrizId { get; set; }
    }
}

File: src/OnPeople/FrontEnd/OnPeople-App/src/app/models/empresas/Empresa.ts
import { Departamento } from "../departamentos/Departamento";
import { EmpresaConta } from "./EmpresaConta";


export interface Empresa {
  id: number;
  nomeEmpresa: string;
  nomeFantasia: string;
  sigla: string;
  ativa: Boolean;
  dataCadastro: Date;
  dataDesativacao?: Date;
  filial: Boolean;
  padraoEmail: string;
  logotipo: string;
  matrizId?: number
  empresasContas: EmpresaConta[];
  departamentos: Departamento[];
}
